EventOrganizer/src/EventOrganizer.Blazor/Pages/CreateEvent.razor

```diff
@@ -3,27 +3,29 @@
 @using EventOrganizer.Localization
 @inject IStringLocalizer<EventOrganizerResource> L
 
-<Heading Size="HeadingSize.Is3" Class="text-center">Create Event</Heading>
+<Heading Size="HeadingSize.Is3" Class="text-center mt-5 mb-4">Create Event</Heading>
 
 <Row>
     <Column ColumnSize="ColumnSize.Is6.Is3.WithOffset">
-    <EditForm Model="@Event" OnValidSubmit="Create">
-        <Field>
-            <FieldLabel>@L["Title"]</FieldLabel>
-            <TextEdit @bind-text="@Event.Title"/>
-        </Field>
-        <Field>
-            <FieldLabel>@L["Description"]</FieldLabel>
-            <MemoEdit @bind-text="@Event.Description"/>
-        </Field>
-        <Field>
-            <Check TValue="bool" @bind-Checked="@Event.IsFree">@L["Free"]</Check>
-        </Field>
-        <Field>
-            <FieldLabel>@L["Free"]</FieldLabel>
-            <DateEdit TValue="DateTime" @bind-Date="@Event.StartTime"/>
-        </Field>
-        <Button Type="@ButtonType.Submit" Color="@Color.Primary">@L["Save"]</Button>
-    </EditForm>
+        <div class="p-lg-5 p-md-3 event-form">
+            <EditForm Model="@Event" OnValidSubmit="Create">
```

Comment:
It's better to use Validations instead of EditForm. Look at this https://github.com/abpframework/abp/blob/06e37d703b8ac3494c900c3ff8a0584588100202/modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/RoleManagement.razor#L82